styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -275,11 +279,55 @@ private static boolean isActiveState(RunStateData runStateData) {
     }
   }
 
-  private Response<Backfill> postBackfill(BackfillInput input) {
+  private <T> Optional<Response<T>> validate(RequestContext rc,
```

Comment:
```suggestion
  private <T> Optional<String> validate(RequestContext rc,
```

Reply:
Would get rid of a lot of `Response.forStatus(Status.BAD_REQUEST.withReasonPhrase(`